es/es-dashboards/README.md

```diff
@@ -1,8 +1,8 @@
-## Install, configure and start Kibana
+# Install, configure and start Kibana
 
-### Manual installation
+## Manual installation
 
-Download Kibana from https://www.elastic.co/fr/downloads/kibana. For Geonetwork 3.8.x download at least version 7.2.1
+Download Kibana from https://www.elastic.co/downloads/kibana. For Geonetwork 3.8.x download at least version 7.2.1
```

Comment:
This is referencing GeoNetwork `3.8.x` and probably also and old version of Kibana.

Reply:
I was only working on the documentation formatting, perhaps it is better t remove version numbers from this section.